moto/resourcegroupstaggingapi/models.py

```diff
@@ -509,6 +513,23 @@ def format_tag_keys(
                     continue
                 yield {"ResourceARN": f"{topic.arn}", "Tags": tags}
 
+        
+        # SSM 
+        if not resource_type_filters or "ssm" in resource_type_filters:
+            for document in self.ssm_backend._documents.values():
+                doc_name = document.describe()["Name"]
+                tags = self.ssm_backend._get_documents_tags(doc_name)
+                if not tags or not tag_filter(
+                    tags
+                ): # Skip if no tags, or invalid filter
+                    continue
+                # pytest.set_trace()
```

Comment:
This should be removed

Reply:
hi, i've removed this comment in my latest commit